examples/bench/compare/segment/main.go

```diff
@@ -21,12 +23,12 @@ var (
 	seedBrokers = flag.String("brokers", "localhost:9092", "comma delimited list of seed brokers")
 	topic       = flag.String("topic", "", "topic to produce to or consume from")
 
-	writeSize   = flag.Int("records-per-write", 1, "number of records to create before a single WriteMessage call")
-	batchSize   = flag.Int("batch-size", 1, "value for the kafka.Writer.BatchSize field")
-	acks        = flag.Int("acks", -1, "value for the Writer.Acks field")
-	recordBytes = flag.Int("record-bytes", 100, "bytes per record (producing)")
-	compression = flag.String("compression", "none", "compression algorithm to use (none,gzip,snappy,lz4,zstd, for producing)")
-	poolProduce = flag.Bool("pool", false, "if true, use a sync.Pool to reuse record slices (producing)")
+	batchSize           = flag.Int("batch-size", 10000, "value for the kafka.Writer.BatchSize field")
+	acks                = flag.Int("acks", 0, "value for the Writer.Acks field")
```

Comment:
I'd favor -1 here still; the intent was for this to be a _relatively_ 1:1 comparison and to use recommended defaults.

Reply:
:+1: